drops/arc_windows_esu_dashboard.json

```diff
@@ -0,0 +1,26 @@
+{
+    "Title": "Azure Arc Windows ESU Dashboard",
+    "Summary": "The dashboard provides a comprehensive view of Azure Arc-enabled Windows and Linux servers, showcasing the installed Arc agent count, operating system distribution, agent version details, detected SQL instances, Windows ESU status, and the allowance and status of Azure Arc extensions.",
+    "Cover": "The dashboard provides a comprehensive view of Azure Arc-enabled Windows and Linux servers, showcasing the installed Arc agent count, operating system distribution, agent version details, detected SQL instances, Windows ESU status, and the allowance and status of Azure Arc extensions.",
```

Comment:
"Cover" -> should be "Description" . Also add "Cover" and you can leave that empty

Reply:
Also, Description generally it's a bit more detailed than summary